src/librustdoc/html/static/js/main.js

```diff
@@ -803,15 +803,10 @@ function loadCss(cssUrl) {
         }
     });
 
-    function handleClick(id, f) {
-        const elem = document.getElementById(id);
-        if (elem) {
-            elem.addEventListener("click", f);
-        }
+    const mainElem = document.getElementById(MAIN_ID);
```

Comment:
I think I originally did it this way to prevent having an extra variable around. But doesn't matter, it's simpler in this case.